From d3fbd21ec1b67929c0c40c46c21d01ac2957f1f7 Mon Sep 17 00:00:00 2001
From: Sean Paul <seanpaul@chromium.org>
Date: Tue, 14 Jul 2015 15:43:20 -0400
Subject: [PATCH 2/2] libdrm: Add rotation property fields

From drm_crtc.h, for use with the plane "rotation" property

NOTE: These are legacy definitions for old code, similar
DRM_MODE_* mask values are defined in the upstream code.

Change-Id: I9240dede6a53817e6d8cedb31f6877312078b5ab
Signed-off-by: Sean Paul <seanpaul@chromium.org>
[jstultz: Added note about legacy definitions ]
Signed-off-by: John Stultz <john.stultz@linaro.org>
---
 include/drm/drm_mode.h | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/include/drm/drm_mode.h b/include/drm/drm_mode.h
index 5fe6c649..3a78de05 100644
--- a/include/drm/drm_mode.h
+++ b/include/drm/drm_mode.h
@@ -204,6 +204,19 @@ extern "C" {
 		DRM_MODE_REFLECT_X | \
 		DRM_MODE_REFLECT_Y)
 
+
+/*
+ * Legacy definitions for old code that doesn't use
+ * the above mask definitions. Don't use in future code.
+ */
+/* rotation property bits */
+#define DRM_ROTATE_0   0
+#define DRM_ROTATE_90  1
+#define DRM_ROTATE_180 2
+#define DRM_ROTATE_270 3
+#define DRM_REFLECT_X  4
+#define DRM_REFLECT_Y  5
+
 /* Content Protection Flags */
 #define DRM_MODE_CONTENT_PROTECTION_UNDESIRED	0
 #define DRM_MODE_CONTENT_PROTECTION_DESIRED     1
-- 
2.35.1

